source/administration/image-proxy.rst

```diff
@@ -3,24 +3,21 @@
 Image Proxy
 ================================
 
-Using image proxies means that clients make only secure requests. By allowing image requests to go straight to third-party
-servers, tracking pixels is allowed. Users can put invisible images in posts that logs time and location data
-for every user that views the post. An image proxy protects user privacy by eliminating this direct interaction with 
-third-party servers.
+Using an image proxy makes it so that all requests for images made by Mattermost clients will go through the proxy instead of contacting third-party servers directly. This helps protect user privacy by preventing third-party servers from tracking who views an image. This also prevents the use of tracking pixels, invisible images that do the same thing without the user even seeing an image.
 
-Proxy servers also provide a layer of caching, and can be made faster and more reliable than third-party sites. This caching 
+Certain proxy servers also provide a layer of caching which can make loading images faster and more reliable. This caching 
 also helps preserve posts by protecting them from dead images.
 
 When enabled, the image proxy needs to be publicly accessible to both the Mattermost client and server.
 
-The Mattermost clients will use the image proxy to load all external images. The Mattermost server will use the image proxy when possible, but it will not use it when requesting content that may not be an image, such as for `image previews of plaintext URLs <https://github.com/mattermost/mattermost-server/issues/11857>`_.
+Mattermost clients will use the image proxy to load all external images. The Mattermost server will use the image proxy when possible, but it will not use it when requesting content that may not be an image, such as for `image previews of plaintext URLs <https://github.com/mattermost/mattermost-server/issues/11857>`_.
 
-You may alternatively use `atmos/camo <https://github.com/atmos/camo>`_ http proxy to route images through SSL:
+An image proxy may be configured from the **Environment > Image Proxy** section of the System Console in recent version of Mattermost or the **Files > Storage** section in versions prior to 5.12.
 
 Local Image Proxy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The local image proxy is enabled on the Mattermost server by default. When using the local image proxy, Mattermost apps will download external images through the Mattermost server itself.
+The local image proxy is available as part of the Mattermost server itself. When using the local image proxy, images will be served to clients through the server which helps anonymize users and provides a secure connection if SSL is enabled on the server. This method does not offer any caching behavior.
```

Comment:
```suggestion
The local image proxy is available as part of the Mattermost server deployment. When using the local image proxy, images are served to clients through the server which helps anonymize users. If SSL is enabled on the server it provides a secure connection. This method does not offer any caching behavior.
```

Reply:
I also added a comma after "If SSL is enabled on the server, ..."